packages/@n8n/nodes-langchain/nodes/chains/ChainLLM/ChainLlm.node.ts

```diff
@@ -88,7 +88,7 @@ async function getImageMessage(
 		NodeConnectionType.AiLanguageModel,
 		0,
 	)) as BaseLanguageModel;
-	const dataURI = `data:image/jpeg;base64,${bufferData.toString('base64')}`;
+	const dataURI = `data:${binaryData.mimeType};base64,${bufferData.toString('base64')}`;
```

Comment:
I wonder if we should throw an error in case mimeType is not an image? 🤔 Other than that LGTM